vnext/Microsoft.ReactNative/Utils/TransformableText.h

```diff
@@ -0,0 +1,55 @@
+#pragma once
+
+#include <boost/algorithm/string.hpp>
+
```

Comment:
There's been some conversation about wanting to use less Boost, as we should eventually be rid of Folly in favor of raw JSI and may be able to remove it. to_upper_copy, etc can be pretty easily implemented without dependencies using std::transform and to_upper, etc. Consider that to prevent future cleanup.

Reply:
I agree in principle, though, Boost is already a hard dependency for React Native.
Depending on the use-case (don't know enough about this one) there are valid scenarios for leveraging boost, as long as everything stays within the ABI boundary.

Reply:
> There's been some conversation about wanting to use less Boost, as we should eventually be rid of Folly in favor of raw JSI and may be able to remove it. to_upper_copy, etc can be pretty easily implemented without dependencies using std::transform and to_upper, etc. Consider that to prevent future cleanup.

I considered that but globalization is hard. For example the uppercase of the German letter ß is SS, so you can't really do letter-by-letter. I'll switch everything to LCMapString though since that makes this more uniform and I don't trust that boost might do the right thing for the locale